site/content/jobs/head-of-marketing.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Head of Marketing
+subtitle: Full time / Remote
+position_state: Open
+draft: false
+workable_code: DC0CA546A4
+---
+
+- Remote & worldwide
+- English
+- Developer-Tools, PLG
+- Full time
+
+## About the role
+
+Checkly is on a mission to build the **#1 reliability automation platform for developers.** We make End-to-End automation delightful, fast, and code-driven. A workflow with a world-class developer experience!
+
+We believe in a bright future where apps are built and shipped more reliably from dev to production and further.
+
+Thousands of developers are using Checkly today, we have recently passed the milestone of 1 billion check runs and raised our $10M USD Series A led by CRV!
+
+We are on a bottoms-up/product-led growth (PLG) path and want to continue growing organically. Our goals are:
+
+1. Bring Checkly to more than 10,000 developers during the next 18 months.
+2. Build a strong community of developers that is rooting for us.
+3. Extent our platform and with that make End-to-End reliability fun!
```

Comment:
"Ship features, extend our platform and make End-to-End reliability fun!"

Reply:
thanks! 'fixed'